snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedConceptSearchRequestBuilder.java

```diff
@@ -93,7 +88,12 @@ public final SnomedConceptSearchRequestBuilder withParsedTerm() {
 	 * @return <code>this</code> search request builder, for method chaining
 	 */
 	public final SnomedConceptSearchRequestBuilder filterByTerm(String term) {
```

Comment:
Pull up the `filterByTerm()` and `filterByExactTerm()` support to the `TermFilterSupport` interface.